TheMET/TheMET/FeaturedArtistsView/FeaturedArtistsView.swift

```diff
@@ -27,13 +27,23 @@ class FeaturedArtistsView: UIView {
         }
     }
     
+    var image: UIImage? {
+        get { return self.imageView.image }
+        set {
+            if let newValue = newValue {
```

Comment:
There is no need to unwrap `newValue` as `self.imageView.image` is also optional